src/dmd/statement.d

```diff
@@ -1610,6 +1610,11 @@ extern (C++) final class SwitchStatement : Statement
                 // All good, the label's scope has no variables
                 return false;
             }
+            else if (vd.storage_class & STC.temp)
+            {
+                // Lifetime ends at end of expression, so no issue with skipping the statement
+                return false;
```

Comment:
That's why I wanted to have coverage info for this.
It looks like the test case is already fixed - even with the latest dmd + ldc release: https://run.dlang.io/is/ZoH9Fs

Reply:
Weird.  Something recent must have changed the logic path.  Shall we just close this and the bug as "Works for me"?